<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSBizLivePusherCtrl</class>
 <widget class="QWidget" name="GBSBizLivePusherCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="dwgtDanmaku" native="true">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>-40</y>
     <width>671</width>
     <height>961</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>453</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F9F9F9;
border: 1px solid  #FFFFFF;
border-radius: 16px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="lblCamera">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>32</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>摄像头驱动</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblMicrophone">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>86</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>麦克风驱动</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxCamera">
     <property name="geometry">
      <rect>
       <x>113</x>
       <y>27</y>
       <width>303</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #676767;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxMicrophone">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>81</y>
       <width>303</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #676767;</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>621</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F9F9F9;
border: 1px solid  #FFFFFF;
border-radius: 16px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>违规提醒</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>200</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#00C566;</string>
     </property>
     <property name="text">
      <string>使用说明</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAgainstRule">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-image: url(:gbs/images/gbs/biz/gbs-widget-docking.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="OBSBasicPreview" name="wgtPreview" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>621</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, 
stop:0 #000000, stop:0.525 #434343, stop:1 #000000); 
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="DoubleTextButton" name="btnStartLive">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>550</y>
      <width>150</width>
      <height>137</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>541</width>
     <height>911</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #01C667;
font-size:16px;
</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::TabShape::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::TextElideMode::ElideLeft</enum>
   </property>
   <widget class="QWidget" name="tabAdvanced">
    <attribute name="title">
     <string>汇总</string>
    </attribute>
    <widget class="QPushButton" name="btnF2F">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>0</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>留言互动</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFans">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>0</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>谁来了</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTips">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>礼物打赏</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>541</width>
       <height>851</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
    <widget class="QPushButton" name="btnAll">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>全部</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabBasic">
    <attribute name="icon">
     <iconset resource="../../forms/obs.qrc">
      <normaloff>:/gbs/images/gbs/biz/gbs-teleprompter.png</normaloff>:/gbs/images/gbs/biz/gbs-teleprompter.png</iconset>
    </attribute>
    <attribute name="title">
     <string>提词器</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OBSBasicPreview</class>
   <extends>QWidget</extends>
   <header location="global">window-basic-preview.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DoubleTextButton</class>
   <extends>QPushButton</extends>
   <header location="global">gbs/common/DoubleTextButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../forms/obs.qrc"/>
 </resources>
 <connections/>
</ui>
